documentation/docs/20-setup/20-local-setup.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Local setup
+---
+
+The STDIO (local) version of the MCP server is available as an `npm` package: `@sveltjs/mcp`. You can either install it globally and then reference it in your configuration or run it with `npx`.
+
+```bash
+npx -y @sveltejs/mcp
+```
+
+Here's how to set it up in some common MCP clients:
+
+## Claude Code
+
+To include the local MCP version in Claude Code, simply run the following command:
+
+```bash
+claude mcp add -t stdio -s [scope] [name] npx -y @sveltejs/mcp
+```
+
+You can choose your preferred `scope` (it must be `user`, `project` or `local`) and `name`.
+
+## Claude Desktop
+
+Open the Developer section in settings, click on `Edit Config`. It will open the folder with a `claude_desktop_config.json` file in it. Edit the file to include the following configuration:
+
+```json
+{
+	"mcpServers": {
+		"[name]": {
```

Comment:
same question, can we just put something here so that the user can copy and paste? could it just be `"svelte"`?